attentive-android-sdk/src/main/java/com/attentive/androidsdk/AttentiveApi.java

```diff
@@ -202,7 +202,7 @@ private HttpUrl.Builder getHttpUrlEventsEndpointBuilder() {
 
     @Nullable
     private String parseAttentiveDomainFromTag(String tag) {
-        Pattern pattern = Pattern.compile("window.__attentive_domain='(.*?).attn.tv'");
+        Pattern pattern = Pattern.compile("='([a-z0-9-]+).attn.tv'");
```

Comment:
do we need to be concerned about spaces after the equals sign?

Reply:
Compiled JS files won't have spaces in variable definitions, so there won't be any spaces unless we drastically restructure the dtag code pre-compilation. I think it's safer to not include them as the match and keep it strict, but happy to discuss more if you disagree.